common/customizable_localization/00_divinity_custom_loc.txt

```diff
@@ -567,34 +567,53 @@ GetActualBishopTitle = {
 		localization_key = ruler_title_name
 	}
 
+	# Warcraft
 	# Religion-By-Religion Titles for Bishops based on Liege's Tier.
-
-	#Paganism
+	#Loa
 	text = {
-		trigger = {
+		trigger = { 
 			liege.highest_held_title_tier >= tier_kingdom
-			is_female = no
+			is_female = yes
+			faith.religion = faith:loa_group.religion
```

Comment:
Try this.
`loa_group` isn't faith, it's religion.
`religion = religion:loa_group`